test/profiles/test.profile

```diff
@@ -1,5 +1,5 @@
-   blacklist /sbin/iptables
```

Comment:
Oops, looking into the file history I see it was intended for the sake of the parser testing.

I reverted changes to test/profiles/test.profile and test/profiles/test2.profile.

Reply:
these are probably intended for testing